pkg/api/api.go

```diff
@@ -1,7 +1,10 @@
 package api
 
+import v1 "github.com/fluxcd/helm-operator/pkg/apis/helm.fluxcd.io/v1"
+
 // Server is the interface that must be satisfied in order to serve
 // HTTP API requests.
 type Server interface {
 	SyncMirrors()
+	SyncMirror(helmRelease *v1.HelmRelease) error
```

Comment:
Given this is the API interface, I do not think it can take the `v1.HelmRelease` as an argument. Instead, this should either take something that is usable in an HTTP API context, or we should not rely on the interface in the `operator.go` but instead use the `GitChartSync`.

Reply:
thanks @hiddeco . yes, +1. let me use gitchartsync.

Reply:
done